editor.planx.uk/src/@planx/components/shared/Preview/Card.tsx

```diff
@@ -48,7 +48,7 @@ const Card: React.FC<Props> = ({
   ...props
 }) => {
   const theme = useTheme();
-  const [path, currentCard] = useStore((state) => [
+  const [path, visibleNode] = useStore((state) => [
```

Comment:
Throughout this PR I've tired to maintain existing variable names when deconstructing state from the store. I think this minimises the number of changes, and hopefully maintains context within the component. For example, sometimes the `currentCard` is referred to as the `node`, `visibleNode` or `currentCard` across differing components.